app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -55,7 +55,7 @@ def update
 
       ATTRIBUTES_FOR_SERIALIZATION = %i[
         id user_id organization_id title slug body_markdown cached_tag_list
-        category classified_listing_category_id category processed_html published
+        classified_listing_category_id processed_html published
```

Comment:
Since we are removing a field here are we concerned about the fact that this change is not really backwards compatible? 

Also, would it be helpful to keep it so the user doesn't need to go look up the classified_listing_category_id if they want to know what the category name is?

Reply:
At one point we kinda said that as long as the API is labeled v0, we're ok with minor breakage. That said, this list just includes the attributes we select from the DB. The serializer adds it back it (it's a model method delegating to the association):

```ruby
json.type_of "classified_listing"

json.extract!(
  listing,
  :id,
  :title,
  :slug,
  :body_markdown,
  :category,
  :classified_listing_category_id,
  :processed_html,
  :published,
)

json.tag_list listing.cached_tag_list
json.tags     listing.tag_list
```

Reply:
OH thanks for the clarification didn't realize this was a db retrieval list!